### PR TITLE
Fix to address database connection error.

### PR DESCRIPTION
**Pull request by uttampawar**
1) Refactored runspec and process_time_based_output by moving matplotlib import in the function. This import is unnecessary when user invoke client with --nograph option.

2) Handle database connection error with appropriate error message instead of generic db server error.

### CODE REVIEW

**Comment by uttampawar** [comment]
@priyankasprakash @egalli Can one of you review this PR please?

**Comment by priyankasprakash** [comment]
LGTM

File: Node-DC-EIS-client/process_time_based_output.py [hunk 1]
```diff
@@ -231,6 +228,10 @@ def post_process(temp_log,output_file,results_dir,interval,memlogfile,no_graph):
   processed_file.flush()
 
   if not no_graph:
+    import matplotlib
+    matplotlib.use(matplotlib.get_backend())
```
Comment (egalli) [inline Node-DC-EIS-client/process_time_based_output.py:232]: Out of curiosity, why are setting the backend to the current backend? Does it reset/fix a bug somewhere?

Comment (priyankasprakash) [reply Node-DC-EIS-client/process_time_based_output.py:232]: Yes, it was throwing an error on few machines saying the backend display environment has not been set.

Comment (uttampawar) [reply Node-DC-EIS-client/process_time_based_output.py:232]: Does this look okay to merge?

**Pull request merged**